lib/ansible/modules/cloud/amazon/iam_server_certificate_facts.py

```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/python
-# Copyright: Ansible Project
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
-from __future__ import absolute_import, division, print_function
-__metaclass__ = type
 
+# Copyright: (c) 2018, Allen Sanabria (@linuxdynasty)
```

Comment:
The copyright statement is set to `Ansible Project` for a reason, we maintain author information in the metadata instead.